app/scripts/lib/auth-errors.js

```diff
@@ -116,6 +116,22 @@ define(function (require, exports, module) {
       errno: 127,
       message: t('Invalid authorization code')
     },
+    INVALID_PHONE_NUMBER: {
+      errno: 129,
+      message: t('Phone number invalid')
```

Comment:
I notice the other messages of this type all begin `Invalid...` wherease this ones finishes `...invalid`. Not sure if we're interested in consistency?